filesystem/src/main/java/org/frankframework/filesystem/ftp/FTPFileRef.java

```diff
@@ -80,7 +80,10 @@ private void setFolder(String folder) {
 	@Override
 	public String getName() {
 		String prefix = folder != null ? folder + "/" : "";
-		return prefix + super.getName();
+		if (org.springframework.util.StringUtils.hasLength(prefix) || super.getName() != null) {
```

Comment:
Graag dezelfde API blijven gebruiken als we overal al gebruiken:

```suggestion
		if (StringUtils.isNotEmpty(prefix) || super.getName() != null) {
```

Reply:
Opgelost